com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/HDRenderPipeline.ScreenSpaceGlobalIllumination.cs

```diff
@@ -41,7 +41,7 @@ IndirectDiffuseMode GetIndirectDiffuseMode(HDCamera hdCamera)
             if (hdCamera.frameSettings.IsEnabled(FrameSettingsField.SSGI))
             {
                 var settings = hdCamera.volumeStack.GetComponent<GlobalIllumination>();
-                if (settings.enable.value)
+                if (settings.enable.value && hdCamera.colorPyramidHistoryIsValid && !hdCamera.isFirstFrame)
```

Comment:
RTGI doesn't need the color pyramid, so this should only be false for SSGI

Reply:
Fixed